### PR TITLE
Facebook data store

### PR DESCRIPTION
**Pull request by csuksangium**

### CODE REVIEW

**Review by janelljchen — APPROVED** [review]

File: testapp/models.py [hunk 1]
```diff
@@ -9,10 +11,24 @@ class Restaurants(models.Model):
...
   term = models.CharField(max_length=15)
 
   def __str__(self):
-    return (self.location, self.term)
\ No newline at end of file
+    return (self.location, self.term)
+
+  
+
+
```
Comment (freddiev4) [inline testapp/models.py]: Nit-pick but can we remove this extra whitespace?

File: testapp/views.py [hunk 2]
```diff
@@ -17,6 +17,12 @@
 import urllib
 import yaml
 
+import facebook
+
+#graph = facebook.GraphAPI(access_token=myToken, version=2.10)
+#profile = graph.get_object('me')
+#args = {'fields' : 'address'}
```
Comment (freddiev4) [inline testapp/views.py]: Do we need this bit that commented out? If not can we remove it?

File: testapp/views.py [hunk 3]
```diff
@@ -195,5 +204,36 @@ def yelp_call(keyTerm, location):
...
+            if k == 'hometown':
+                user_info.hometown = fb_user_info['hometown']['name']
+            if k == 'friends':
+                user_info.num_friends = fb_user_info['friends']['summary']['total_count']
+            if k == 'name':
+                user_info.name = fb_user_info['name']
+            if k == 'age_range':
+                user_info.age_range = fb_user_info['age_range']['min']
+            if k == 'religion':
+                user_info.religion = fb_user_info['religion']
```
Comment (freddiev4) [inline testapp/views.py]: I think there might be a better approach to this, like traversing the JSON in some other way but we can come back to this for the sake of time

File: testapp/views.py [hunk 4]
```diff
@@ -195,5 +204,36 @@ def yelp_call(keyTerm, location):
...
+    user_info.save()
+    
+
+#def get_facebook_account():
+    #user =  SocialAccount.objects.get(uid="10210039294274709")
+    #access_token = SocialToken.objects.filter(account__user=user.user, account__provider='facebook').first()
```
Comment (freddiev4) [inline testapp/views.py]: Can we remove this? Or are we still saving data this way?

**Review by freddiev4 — CHANGES_REQUESTED** [review]

**Comment by freddiev4** [comment]
Also looks like there's some merge conflicts here; once you address the other comments I made I can go through the conflict stuff

**Comment by csuksangium** [comment]
did the formatting stuff and ill look into a cleaner way to parse the json

**Review by freddiev4 — APPROVED** [review]

**Pull request merged**